paper.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'BinderHub: shareable, interactive computing environments with Kubernetes'
+tags:
+  - jupyter
+  - jupyterhub
+  - interactive computing
+  - cloud computing
+  - binder
+  - kubernetes
+  - helm
+
+authors:
+ - name: Project Jupyter
+   orcid:
+   affiliation:
+ - name: Tim Head
+   orcid: XXXX
+   affiliation: Wild Tree Tech
+ - name: Jessica Forde
+   orcid: XXXX
+   affiliation: Project Jupyter
+ - name: Brian Granger
+   orcid: 0000-0002-5223-6168
+   affiliation: Cal Poly
+ - name: Chris Holdgraf
+   orcid: 0000-0002-2391-0678
+   affiliation: University of California, Berkeley
+ - name: M Pacer
+   orcid: XXX
+   affiliation: University of California, Berkeley
+ - name: Yuvi Panda
+   orcid: XXXX
+   affiliation: University of California, Berkeley
+ - name: Fernando Perez
+   orcid: XXXX
+   affiliation: University of California, Berkeley
+ - name: Min Ragan-Kelley
+   orcid:  0000-0002-1023-7082
+   affiliation: Simula Research Laboratory
+ - name: Carol Willing
+   orcid: 0000-0002-9817-8485
+   affiliation: Cal Poly
+date: YYYY-MM-DD
+bibliography: paper.bib
+---
+
+# Summary
+
+BinderHub enables authors to create sharable, interactive
+computational environments by specifying their environment in an online
+git repository.
+
+BinderHub runs a service that lets users put their code in a repository online, use
+the BinderHub service to build a Docker image using configuration files in that
+repository, then serve that Docker image to users via a public link. BinderHub
+will flexibly create cloud resources as users request to interact with a
+particular repository's link, and will automatically destroy these resources
+after periods of inactivity. A public deployment of BinderHub exists at
+`mybinder.org`, though BinderHub is designed to be deployed by anyone. It is
+cloud-agnostic and can support many different workflows.
```

Comment:
Even though binder is a very generic way of hosting/deploying a wide range of things, I think it will help us to be a bit more explicit about the main focus - of enabling Jupyter notebooks to be run by anyone anywhere. We should also mention other things (RStudio) etc. as well. 

We should also connect Binder to the broader goals of reproducibility.

Reply:
I am more than willing to take an editing pass at some point if that would help.

Reply:
@ellisonbg I think we've explicitly tried to make sure (and message!) that Binder is not Jupyter specific, so it'd be good to keep that messaging here in some form.

Reply:
incoming push tries to clarify specific use-cases a bit more, LMKWYT when it lands